src/Tags/Range.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Statamic\Tags;
+
+use Statamic\Tags\Tags;
+
+class Range extends Tags
+{
```

Comment:
I now see why it was called `loop` before.
`{{ loop times="5" }}` reads nicer than `{{ range to="5" }}`.

You can alias it by adding `protected static $aliases = ['loop'];` here.

Reply:
Would you like me to rename the class too? It might be clearer if someone is looking for it in the future.

Reply:
Range is still good though. `{{ range from="5" to="10" }}` reads just as nicely.

Reply:
Cool